systemFlake.nix

```diff
@@ -16,7 +16,7 @@
 , sharedExtraArgs ? { }
 , sharedModules ? [ ]
 
-, packagesBuilder ? null
+, packagesBuilder ? (flake-utils-plus.lib.builder.packagesFromOverlayBuilderConstructor channels)
```

Comment:
Not sure if this should be the default - might confuse some users

Reply:
Not sure either, will remove it for now. Being conservative might appeal to our audience.